packages/components/src/components/tables/es-table-detail/es-table-detail.tsx

```diff
@@ -60,7 +76,7 @@ export class TableDetail {
                                 }}
                             >
                                 {this.renderHeader(title)}
-                                {this.renderCell(name, cell)}
+                                {this.renderCell(name)}
```

Comment:
Probably simpler to add  it back here:
Probably simpler to add it here, and remove the `renderHeader` func.

```suggestion
                                <dt>{title}</dt>
                                <dd>{this.renderCell(name)}</dd>
```